js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+advanced=Erweitert
+fineGrainOpenIdConnectConfiguration=Feinkörnige OpenID Connect-Konfiguration
+fineGrainOpenIdConnectConfigurationHelp=In diesem Abschnitt werden die erweiterten Einstellungen dieses Clients für das OpenID Connect-Protokoll konfiguriert.
+openIdConnectCompatibilityModes=OpenID Connect Kompatibilitätsmodi
+openIdConnectCompatibilityModesHelp=Dieser Abschnitt wird verwendet, um Einstellungen für die Abwärtskompatibilität mit älteren OpenID Connect / OAuth 2-Adaptern zu konfigurieren. Dies ist besonders nützlich, wenn Ihr Client eine ältere Version von Keycloak / RH-SSO-Adapter verwendet.
+advancedSettings=Erweiterte Einstellungen
+advancedSettingsOpenid-connect=Dieser Abschnitt wird verwendet, um erweiterte Einstellungen dieses Clients in Bezug auf das OpenID Connect-Protokoll zu konfigurieren
+authenticationOverridesHelp=Authentifizierungsflussbindungen des Realms überschreiben.
+downloadAdapterConfig=Adapterkonfiguration herunterladen
+emptyAdminEvents=Keine Admin-Ereignisse
+emptyAdminEventsInstructions=In diesem Realm sind keine Admin-Ereignisse vorhanden.
+selectALocale=Sprache wählen
+requiredActionPlaceholder=Benutzeraktion auswählen
+credentialUserLabel=Benutzer-Label
+credentialData=Daten
+showDataBtn=Daten anzeigen
+resetPasswordConfirmation=Passworts Bestätigung
+resetPasswordFor=Passwort zurücksetzen für {{username}}
+roleMapping=Rollenzuordnung
+inherent=Geerbt
+noGroups=Keine Gruppen
+noGroupsText=Sie haben diesen Benutzer zu keiner Gruppe hinzugefügt. Treten Sie einer Gruppe bei, um loszulegen.
+joinGroup=Gruppe beitreten
+noConsents=Keine Einwilligungen
+noConsentsText=Die Einwilligungen werden nur aufgezeichnet, wenn Benutzer versuchen, auf einen Client zuzugreifen, der so konfiguriert ist, dass eine Einwilligungen erforderlich ist. In diesem Fall erhalten die Nutzer eine Seite mit der Aufforderung, den Zugriff auf den Client zu genehmigen.
+identityProviderLinks=Identitätsanbieter-Links
+linkedIdPs=Verknüpfte Identitätsanbieter
+linkedIdPsText=Die Identitätsanbieter, die bereits mit diesem Benutzerkonto verknüpft sind
+noProvidersLinked=Keine Identitätsanbieter verknüpft. Wählen Sie einen aus der Liste unten.
+availableIdPs=Verfügbare Identitätsanbieter
+availableIdPsText=Alle konfigurierten Identitätsanbieter in diesem Bereich werden hier aufgelistet. Sie können das Benutzerkonto mit jedem der IdP-Konten verknüpfen.
+noAvailableIdentityProviders=Keine verfügbaren Identitätsanbieter.
+impersonate=Impersonieren
+logoutAllSessions=Alle Sitzungen abmelden
+started=Gestartet
+userCredentialsHelpText=Mit den Handlern der obersten Ebene können Sie die Priorität der Credentials für den Benutzer ändern, wobei das oberste Credential die höchste Priorität hat. Mit den Handlern innerhalb eines erweiterbaren Bereichs können Sie die visuelle Reihenfolge der Credentials ändern; das oberste Credential wird ganz links angezeigt.
```

Comment:
Not really clear for me what this means ;)

Reply:
Yes, that's even hard to understand in the original english translation.
The UI shows this text at the "drag icons" in the user credentials tab
![image](https://github.com/keycloak/keycloak/assets/314690/3d32b5a3-9e6e-487e-91e3-3094f3cbc99c)

I think this is easier to understand:
```
userCredentialsHelpText=Mit den ziehbaren Flächen in der ersten Spalte einer Credentialzeile können Sie die Priorität der Credentials für den Benutzer ändern, wobei das oberste Credential die höchste Priorität hat.
```

I personally also don't get the last part: "Mit den Handlern innerhalb eines erweiterbaren Bereichs können Sie die visuelle Reihenfolge der Credentials ändern; das oberste Credential wird ganz links angezeigt."
Original text:
```
userCredentialsHelpText=... The handlers within one expandable panel allow you to change the visual order of the credentials, the topmost credential will show at the most left.
```
Perhaps this is from an old ui?


So I will keep it as: 
```
userCredentialsHelpText=Mit den ziehbaren Flächen in der ersten Spalte einer Credentialzeile können Sie die Priorität der Credentials für den Benutzer ändern, wobei das oberste Credential die höchste Priorität hat.
```

Reply:
Btw. I need to adjust the text again, as it is possible to group credentials and the user can reorder child credentials within a group:
![image](https://github.com/keycloak/keycloak/assets/314690/e1b34313-56e2-41a0-ad97-324bd26bc5d0)